File: LeetCode/c/598.c
int maxCount(int m, int n, int** ops, int opsSize, int* opsColSize){
    for (int i = 0; i < opsSize; ++i) {
        m = m < ops[i][0] ? m :ops[i][0];
        n = n < ops[i][1] ? n :ops[i][1];
    }
    return m * n;
}

File: LeetCode/python/598.py
class Solution(object):
    def maxCount(self, m, n, ops):
        """
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        """
        for o in ops:
            m = min(m, o[0])
            n = min(n, o[1])
        return m * n